src/context_dictionary.py

```diff
@@ -1,5 +1,5 @@
-from collections import Counter
-from collections import defaultdict
+# from collections import Counter
```

Comment:
Go ahead and delete all these commented lines of old code. We're using GitHub so we can review these changes if we ever need to.